docs/getting-started.mdx

```diff
@@ -5,38 +5,152 @@ description: Quick start guide for Xata's PostgreSQL platform
+WITH o AS (INSERT INTO orders DEFAULT VALUES RETURNING id) INSERT INTO order_items(order_id,product_id,qty) SELECT o.id,pid,(1+floor(random()*3))::int FROM o,(SELECT id pid FROM products ORDER BY random() LIMIT 5)p;
+```
+
+![Queries/SQL editor page](assets/images/getting-started/queries.png)
+
+## 5. View your branch schema and enable schema history
+
+Go to the **Branches** section and select your branch (e.g., `main`). Here you can view your schema and enable schema history for zero-downtime migrations.
+
+To enable schema history, install the CLI and initialize your branch:
+
+```bash
+curl -fsSL https://xata.io/install.sh | bash
+xata auth login
+xata init
+xata roll init
+```
+
+![Branch details/schema view](assets/images/getting-started/branch-details.png)
+
+## 6. Create a development branch with the CLI
+
+List your branches:
+
+```bash
+xata branch list
+```
+
+Grab the `main` branch ID from the list and create a new branch from `main`:
+
+```bash
+ xata branch create --name dev --parent-branch [main branch id]
+```
+
+## 7. Modify your schema from the terminal
+
+You can make schema changes directly in the Xata query view, but here we'll show how to do it from your terminal using `psql` for a more hands-on workflow.
+
+If you don't have `psql` installed:
+
+**macOS:**
+```bash
+brew install libpq
+echo 'export PATH="/opt/homebrew/opt/libpq/bin:$PATH"' >> ~/.zshrc
+source ~/.zshrc
+```
+
+**Linux:**
+```bash
+sudo apt-get install postgresql-client
+```
+
+**Windows:**
+Download from [PostgreSQL official site](https://www.postgresql.org/download/windows/).
+
+Get your branch connection string using the Xata CLI. First, checkout your new `dev` branch:
+
+```bash
+xata branch checkout dev
+```
+Then grab the connection string:
+
+```bash
+xata branch url
+```
+
+Connect to your branch with psql:
+
+```bash
+psql <paste-connection-string-here>
+```
+
+Add a new column to your `products` table:
+
+```sql
+ALTER TABLE products ADD COLUMN rating INT;
+```
+
+You can now use SQL to update or query the new column.
+
+## 8. Check the schema diff in the branch details page
+
+After making changes in a branch, use the **Schema Diff** feature in the `dev` branch details page to compare your branch with its parent.
+
+![Schema diff page](assets/images/getting-started/schema-diff.png)
+
+## 9. Start from a clean slate
+
+Now that you're all set up, let's get you reset and ready to start building.
+
+First, switch to your main branch:
+
+```bash
+xata branch checkout main
+```
+
+Then, delete the `dev` branch:
+
+```bash
+xata branch delete dev
+```
+
+Finally, connect to your main branch using `psql` and run the following SQL to remove the sample tables and data:
+
+```sql
+DROP TABLE IF EXISTS order_items;
+DROP TABLE IF EXISTS orders;
+DROP TABLE IF EXISTS products;
+```
+
+This will leave your main branch clean for your next project or migration.
```

Comment:
Clean, but with schema history... Might be better to just start fresh with a new project.